### PR TITLE
update react-router example to use react-router 1.0

### PR DESCRIPTION
**Pull request by hfter789**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by mridgway** [comment]
Nice. Thanks for getting this started.

File: examples/react-router/server.js [hunk 1]
```diff
@@ -12,39 +12,49 @@ var React = require('react');
...
 
 server.use(function (req, res, next) {
-    var context = app.createContext();
 
-    debug('Executing navigate action');
-    Router.run(app.getComponent(), req.path, function (Handler, state) {
-        context.executeAction(navigateAction, state, function () {
-            debug('Exposing context state');
-            var exposed = 'window.App=' + serialize(app.dehydrate(context)) + ';';
+	debug('Executing navigate action');
```
Comment (mridgway) [inline examples/react-router/server.js]: Could you switch to using spaces instead of tabs in this file?

File: examples/react-router/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
   "dependencies": {
     "fluxible": "^0.5.0",
     "fluxible-addons-react": "^0.1.6",
+    "history": "^1.17.0",
     "react": "^0.13.0",
-    "react-router": "^0.13.1"
+    "react-router": "^1.0.0"
```
Comment (mridgway) [inline examples/react-router/package.json]: We should move these dependencies to the [parent directory's package.json](https://github.com/yahoo/fluxible/blob/master/examples/package.json#L22). This way there isn't an additional install needed for this example anymore.

Comment (hfter789) [inline examples/react-router/package.json]: Good point. 
After I moved the dependencies under examples/package.json, react-router server cannot correctly match the url, and there is no error. I will investigate more later.

File: examples/package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "fluxible-addons-react": "^0.2.0",
     "fluxible-plugin-fetchr": "^0.3.3",
     "fluxible-router": "^0.4.0",
+    "history": "^1.17.0",
     "react": "0.14.x",
     "react-dom": "0.14.x",
     "react-router": "^1.0.0-rc3",
```
Comment (mridgway) [inline examples/package.json]: Update to `^1.0.3` here. I'm seeing weird issues with this where it's installing `1.0.2` and throwing peer invalid errors on installation.

**Comment by hfter789** [comment]
@mridgway Changed the version number, but the problem still exists.

**Comment by hfter789** [comment]
@mridgway This PR is ready for review. And thank you for all the help.

**Comment by mridgway** [comment]
:+1:

**Comment by redonkulus** [comment]
:+1:

**Comment by mridgway** [comment]
Forgot to say, thanks a bunch @hfter789!

**Pull request merged**